Lib/unittest/test/test_case.py

```diff
@@ -577,15 +577,15 @@ def testShortDescriptionWithoutDocstring(self):
     @unittest.skipIf(sys.flags.optimize >= 2,
                      "Docstrings are omitted with -O2 and above")
     def testShortDescriptionWithOneLineDocstring(self):
-        """Tests shortDescription() for a method with a docstring."""
+        """ Tests shortDescription() for a method with a docstring."""
```

Comment:
Minor: This contradicts standard docstring style and muddies git blame, please revert.